package.json

```diff
@@ -23,14 +23,14 @@
   },
   "homepage": "https://github.com/vieron/stylelint-webpack-plugin#readme",
   "peerDependencies": {
-    "stylelint": "^7.0.1"
+    "stylelint": "^7.0.1",
+    "webpack": "^2 || ^2.2.0-rc.0 || ^2.1.0-beta.0 || ^1.13.2"
```

Comment:
I see, because the `rc` and `beta` versions do not fall under semver technically so they aren't resolved by `^2.0.0`? Fair enough.